From: Miaohe Lin <linmiaohe@huawei.com>
Subject: mm: remove obsolete alloc_migrate_target()
Date: Sat, 3 Jun 2023 22:25:13 +0800

There's only declaration left in the header file. Remove it.

Link: https://lkml.kernel.org/r/20230603142513.787000-1-linmiaohe@huawei.com
Signed-off-by: Miaohe Lin <linmiaohe@huawei.com>
Reviewed-by: David Hildenbrand <david@redhat.com>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/page-isolation.h |    3 ---
 1 file changed, 3 deletions(-)

--- a/include/linux/page-isolation.h~mm-remove-obsolete-alloc_migrate_target
+++ a/include/linux/page-isolation.h
@@ -45,7 +45,4 @@ void undo_isolate_page_range(unsigned lo
 
 int test_pages_isolated(unsigned long start_pfn, unsigned long end_pfn,
 			int isol_flags);
-
-struct page *alloc_migrate_target(struct page *page, unsigned long private);
-
 #endif
_
